### ISSUE TITLE
Problems if review comments have quotes

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1496]
Either @peller or @aerwin appeared to have added a comment during testing that included a double-quote and this causes the JSON parsing of the comments data structure to fail when I tried to reopen. It looks like the R/C code isn't escaping quotes properly.

**Comment by JonFerraiolo** [comment]
@EldonReturn, @waynevicknair

**Comment by aerwin** [comment]
I think that must have been my comment.. I wrote the following:

   I think it should be "Remove"

When I go back in, I can see my comments, but the quotes are gone.

**Comment by JonFerraiolo** [comment]
OK, I figured out what's going wrong by studying the JSON response in the network panel. Basically, if you include a double-quote in a text annotation, you'll get erroneous behavior. The reason is that the text annotation is stored as a double-JSON-encoded syntax. Here is a snippet that shows the problem (I added spaces and newlines for readability):

```
drawingJson : 
"
[
    {
    type:\"Rectangle\",
    x1:250,y1:25,x2:300,y2:81,
    colorAlias:\"aerwin@us.ibm.com\",
    commentId:\"173349ac-d6d3-6f0a-49d1-1c24e84849b3\",
    state:\"Normal\"
    },
    {
    type:\"Text\",
    x1:295,y1:8,x2:391,y2:41,
    colorAlias:\"aerwin@us.ibm.com\",
    commentId:\"173349ac-d6d3-6f0a-49d1-1c24e84849b3\",
    state:\"Normal\",
    text:\"Make this \"Remove\"\"
    }
]
",
```

I suspect the problem is that there needs to be two levels of escaping quotes (in case there are any quotes within the "text" property) but there is only one level right now.

**Comment by JonFerraiolo** [comment]
OK, I've got a fix for this bug. I'll commit is in within a couple of hours.

**Comment by JonFerraiolo** [comment]
Fixed by a268dcec4767799c125eaebd85167dd4f81c68c4
